src/edit/main.js

```diff
@@ -161,6 +157,20 @@ class ProseMirror {
     else if (opts.place)
       opts.place(this.wrapper)
 
+    // does not support opts.place being a function and asynchronously mount prosemirror
+    this.root = this.wrapper.parentNode
+    if (this.root) {
+      // loop if root is not document (in light dom)
+      // only shadowRoot has property (in shadow dom)
+      while (this.root!== document && !this.root.host) {
+        this.root = this.root.parentNode
+      }
+    } else
+      this.root = document
+
+    require("./css")(this)
```

Comment:
Will this mean it's added multiple time to a root, if there are multiple instance of ProseMirror in the same root?